examples/landing-zone-v2/configconnector/tier3/dns/dns.yaml

```diff
@@ -19,7 +19,7 @@ kind: DNSRecordSet
 metadata:
   name: workload-name-client-name-standard-core-public-dns-a-rset # kpt-set: workload-name-${client-name}-standard-core-public-dns-a-rset
   annotations:
-    cnrm.cloud.google.com/project-id: host-project-id # kpt-set: ${host-project-id}
```

Comment:
I'm not sure about this change, as I would probably enter the core dns project id instead. I think I made that mistake last week when deploying a dev LZ.

Reply:
Never mind, I see that the client will get a dedicated dns project.